### ISSUE TITLE
We are unable to use localstorage, is there a way to save response to scorm?

### ISSUE THREAD

**Issue by robmausser** [LearningPool/adapt-contrib-openTextInput#37]
We are unable to use localstorage because many users will use the same machine.

Is there a way to instead set storeUserAnswer to store the inputted text to scorm and then retrieve it again when the learner opens the course?

**Comment by brian-learningpool** [comment]
Hi @robmausser, apologies for the late reply.  By default the component doesn't use SCORM due to the limitations on the number of characters which can be stored in SCORM 1.2.

**Comment by brian-learningpool** [comment]
Hi @robmausser, that's correct, this currently isn't implemented.  If you feel up to it though I would accept a PR to optionally store this the LMS.

**Comment by demonking** [comment]
@robmausser @brian-learningpool something new for this issue?

**Comment by moloko** [comment]
> Meaning its impossible with SCORM 1.2 to retrieve a student response back from the LMS, at all, period.
> 
> Can you confirm this?

Yes, that's right. The `cmi.interactions` data fields are all marked as write-only in the SCORM spec. They are for the purposes of reporting what the learner did - not for persisting session state.

**Comment by paul-mediakitchen** [comment]
I notice the default _allowedCharacters is 500 - if I understand correctly, SCORM 1.2 will only record 255 characters in interaction data so if I am right about this I wonder if 255 would be a better default? I may be wrong of course.

**Comment by moloko** [comment]
@paul-mediakitchen makes a good point, `cmi.interactions.n.student_response` is of type `CMIFeedback` where the 'fill-in' datatype is limited to 255 characters:
![image](https://user-images.githubusercontent.com/1249597/93066093-730a1780-f671-11ea-8804-93857e5635ea.png)
